Resources/Textures/Interface/Misc/health_icons.rsi/meta.json

```diff
@@ -5,10 +5,29 @@
         "y": 8
     },
     "license": "CC-BY-SA-3.0",
-    "copyright": "https://github.com/tgstation/tgstation/blob/master/icons/mob/huds/hud.dmi",
+    "copyright": "Taken from https://github.com/tgstation/tgstation/commits/master/icons/mob/huds/hud.dmi and modified by DoutorWhite",
```

Comment:
This needs to reference the commit it was taken from.

Reply:
I think I solved both issues